client/hotrod-client/src/test/java/org/infinispan/client/hotrod/configuration/ConfigurationTest.java

```diff
@@ -193,7 +193,7 @@ public void testConfiguration() {
             .testWhileIdle(false)
             .timeBetweenEvictionRuns(15000)
             .connectionTimeout(100)
-            .version(ProtocolVersion.PROTOCOL_VERSION_13)
+            .version(ProtocolVersion.PROTOCOL_VERSION_29)
```

Comment:
We should probably try a version that isn't the latest/default. Maybe 2.6?

Reply:
I put it at 2.9 because I'll soon have a PR that bumps the protocol to 3.0